### ISSUE TITLE
[no-jasmine-globals] Verify Jasmine pending() global usage

### ISSUE THREAD

**Issue by macklinu** [jest-community/eslint-plugin-jest#156]
_Originally posted by @SimenB in https://github.com/jest-community/eslint-plugin-jest/pull/155#issuecomment-423440207_

> @macklinu I wonder if https://github.com/jest-community/eslint-plugin-jest/blob/master/docs/rules/no-jasmine-globals.md might need the same treatment that `pending` got in this PR.

This issue is a follow-up to check that the `no-jasmine-globals` rule doesn't have the same issues reported in #149.

**Comment by jdelStrother** [comment]
I think I'm running into basically this problem.  With a parameter named `fail`: 

```js
function foo(success, fail) {
  fail();
}
```

I get "Illegal usage of `fail`, prefer throwing an error, or the `done.fail` callback`"

**Comment by SimenB** [comment]
Yeah, definitely! Wanna send a PR? You can see #155 for how to implement it 🙂
